ckanext/ontario_theme/templates/internal/group/read.html

```diff
@@ -69,22 +63,20 @@
 
   {% block secondary_content %}
     <div class="ministry-package-div secondary col-sm-3">
-      <div class="filters" id="filter-aside">
+      <div class="filters" id="filter-aside" data-module="facet_checkboxes">
         <span aria-label="{{ _('close') }}"
               role="button"
               tabindex="0"
               class="close no-text hide-filters">
           <svg class="ontario-icon hide-sort-by"
-              focusable="false"
-              sol:category="primary"
-              viewBox="0 0 24 24"
-              preserveAspectRatio="xMidYMid meet">
+               focusable="false"
+               sol:category="primary"
+               viewBox="0 0 24 24"
+               preserveAspectRatio="xMidYMid meet">
             <use xlink:href="#ontario-icon-close"></use>
           </svg>
         </span>
-        <div>
-          {{ h.snippet('snippets/facet_arrangement.html', single_option_facets=single_option_facets, extras={'id':c.group_dict.id}) }}
-        </div>
+        <div>{{ h.snippet('snippets/facet_arrangement.html', extras={'id':c.group_dict.id}) }}</div>
```

Comment:
In `organization/read.html`, you pass `single_option_facet="organization"` to `facet_arrangement.html`:

```
{{ h.snippet('snippets/facet_arrangement.html', extras={'id':c.group_dict.id}, 
single_option_facet="organization") }}
```
I'm wondering if you need to do the same for groups. Otherwise, all the groups will be listed in the facet column of an individual group page. 

<img width="319" alt="Screenshot 2024-09-17 at 18 22 50" src="https://github.com/user-attachments/assets/cd04c7c2-d60a-4596-9ae2-a254d4e6b7e8">
(screenshot from local)